packages/airnode-protocol/contracts/authorizers/mock/MockRequesterAuthorizerWithErc721.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import "../../dev/RequesterAuthorizerWithErc721.sol";
```

Comment:
I wonder if we want to copy the contents to the Airnode repo or use it as a dependency. Dependency soundsw better to me.

Reply:
Burak suggested copy the contents:https://api3workspace.slack.com/archives/C0436H2BJKE/p1677057747622019?thread_ts=1676612819.041809&cid=C0436H2BJKE

Reply:
I can temporarily publish https://github.com/api3dao/nft-authorizer separately if you want, but to me these feel like equivalent solutions

Reply:
I was convinced by the argument that these will eventually be in this repo anyway.

Reply:
Yeah, if it's going to be moved I am ok with it. An issue in the Airnode repo would be nice though.

Reply:
https://github.com/api3dao/airnode/issues/1675